demos-europe/demosplan-core
# fix(refs T34297): Use Permission 'area_admin_boilerplates' in FE

**Ticket:** https://yaits.demos-deutschland.de/T34297



If the Permission is dispabled, there shouldn't be a boilerplate modal/button in the editor

:warning:  Is the assumption correct, that a user, that can insert biolerplates can also create them? 


### How to review/test


Find a Project with "area_admin_boilerplates" enabled. Check, that You can use boilerplates like a charm. Then disable "area_admin_boilerplates" for Your current user and test if the boilerplates are gone without trowing any FE-Errors

e.g. as FPA `/verfahren/<procedure-id>/abwaegung/sview/<stn-id>`
and `/verfahren/testverfahren-flora/textbausteine`


:warning:  We may want to wait until codefreeze because its not necessarily required for this sprint and the changes are targeting several spots

### Linked PRs (optional)


### Tasks (optional)


### PR Checklist


Delete the checkbox if it doesn't apply/isn't necessary.
